### ISSUE TITLE
[Feature request] Multiple accounts support

### ISSUE THREAD

**Issue by rizaumami** [telegramdesktop/tdesktop#4261]
### Steps to reproduce
Nothing.

### Expected behaviour
There is menu to login into (three?) different accounts just like in Telegram v4.7 for Android, so I don't have to open multiple Telegram Desktop windows and messing with launcher script or config directory, etc.

### Actual behaviour
I can now login into multiple accounts by issuing `-workdir` or `-many -workdir`, but this is an inconvenient process because I have to open multiple Telegram Desktop windows and provide many config directories. 

### Configuration
**Operating system:** Debian Sid 64 bit
**Version of Telegram Desktop:** 1.2.6 alpha
**Used theme**: Default

<details><summary><b>Logs</b>:</summary>
Insert logs here (if necessary)

</details>

**Comment by ArmeF97** [comment]
Try Frantz, it's not perfect but it's something.
I'd love to have it implemented natively, too 👍

**Comment by rizaumami** [comment]
I knew about Franz, Rambox, Wavebox or just the new Firefox Quantum multi container. But that all just a Telegram web (wrapper), which mean it's limited in features and resource hungry. I hope for a native implementation.

**Comment by cobalto** [comment]
#533

**Comment by stek29** [comment]
@cobalto Multiple instances != Multiple accounts in once instance.

**Comment by cobalto** [comment]
@stek29 disagree.
They asked for multiple instances to have multiple accounts at the same machine when "multiple accounts" wasnt a feature of telegram. Now it is and now you can ask for it.

So, I think its for the same use, that's why I quoted the other one here.

But I probably should have done the opposite, but now both are linked and that's what I wanted. Not gonna fix that now ^^'

**Comment by vogelito** [comment]
Would love to see this.

**Comment by Kirill** [comment]
I need this too

**Comment by Fathurhoho** [comment]
I really need this feature.

**Comment by Silicomancer** [comment]
Needed badly here too.

**Comment by zeroone2numeral2** [comment]
_Yes yes same here_

I claimed my heart reaction too, kind stranger. Now deliver

**Comment by azzamsa** [comment]
hope it will be implemented soon. So we can easily switch between account for family and work.

**Comment by hroost** [comment]
+1

**Comment by rikarsen** [comment]
waiting for it

**Comment by ShadowJonathan** [comment]
+1

**Comment by sigaev-pro** [comment]
+1

**Comment by RememberTheAir** [comment]
Stop with the +1s, reactions are there for a reason. +1s are just useless noise

**Comment by avele** [comment]
+1

**Comment by Kirill** [comment]
@john-preston, did you have plan for this issue?

**Comment by Piterden** [comment]
Hi, guys)

Any updates?

**Comment by nectop** [comment]
waiting for implementation.

**Comment by dfmabbas** [comment]
+1

**Comment by Hyd3L** [comment]
It's available on Android since Pleistocene, why it takes so long to do the same thing on the desktop client?

**Comment by sprite-1** [comment]
Hope this gets added soon

**Comment by ange007** [comment]
Very need this function.
It is very important to separate the work account and the personal.

**Comment by amirashabani** [comment]
Any updates on this feature? :disappointed:

**Comment by Banginpadr** [comment]
> ### Steps to reproduce
> 
> Nothing.
> ### Expected behaviour
> 
> There is menu to login into (three?) different accounts just like in Telegram v4.7 for Android, so I don't have to open multiple Telegram Desktop windows and messing with launcher script or config directory, etc.
> ### Actual behaviour
> 
> I can now login into multiple accounts by issuing `-workdir` or `-many -workdir`, but this is an inconvenient process because I have to open multiple Telegram Desktop windows and provide many config directories.
> ### Configuration
> 
> **Operating system:** Debian Sid 64 bit
> **Version of Telegram Desktop:** 1.2.6 alpha
> **Used theme**: Default
> Logs:
> Insert logs here (if necessary)

first go there and download it for you system. 1)TelegramForcePortable https://imgur.com/a/VUGxOrU 2 method) **On macOS,** just use `-workdir` parameter, it won’t work if you add many <path to Telegram binary> -workdir <path to folder>

**Comment by markuznw** [comment]
Subscribing to this, guess tdesktop is the only remaining client between both official clients and unofficial that still don't have it. Hope it gets implemented soon :)

**Comment by Nachtalb** [comment]
I don't know about windows. But on OSX and Linux you can use the `-workdir` argument to create more instances. 

I do know this is inconvenient for some people and that this issue exists for over a year now and there is still no way in the client itself to switch accounts with a few mouse clicks. But neither have I seen some guide to use this feature so here it is. I hope it helps you out. 

:exclamation: replace words in {{}} with your own username and such (without the brackets).

Linux
---
- Create a folder here `~/.local/share/TelegramDesktop/{{MyUsername}}` 
- Create a new `.desktop` file here `~/.local/share/applications/telegramdesktop-{{myusername}}.desktop`. 
- Put in your data in here
```desktop
[Desktop Entry]
Version=1.0
Name=Telegram MyUsername
Comment=Official desktop application for the Telegram messaging service
TryExec=/opt/Telegram/Telegram
Exec=/opt/Telegram/Telegram -workdir /home/{{your_user}}/.local/share/TelegramDesktop/{{MyUsername}} -- %u
Icon=telegram
Terminal=false
StartupWMClass=TelegramDesktop
Type=Application
Categories=Network;InstantMessaging;Qt;
MimeType=x-scheme-handler/tg;
Keywords=tg;chat;im;messaging;messenger;sms;tdesktop;
X-GNOME-UsesNotifications=true
X-Desktop-File-Install-Version=0.23
```

OSX
--- 
- Create a folder here: `~/.local/share/TelegramDesktop/{{MyUsername}}`
- Open Automator
- Create a new Application
![image](https://user-images.githubusercontent.com/9467802/56354968-6875f500-61d5-11e9-9e61-32f87ba5502c.png)
- Add an apple script by drag & drop from the left-hand side
- Add the following text:
```
do shell script "Applications/Telegram.app/Contents/MacOS/Telegram -workdir '/Users/{{your_user}}/.local/share/TelegramDesktop/{{MyUsername}}'"
```
![image](https://user-images.githubusercontent.com/9467802/56354966-657b0480-61d5-11e9-8341-200ee72d1cb2.png)
- Save to `/Applications/Telegram {{MyUsername}}.app`
- Now lastly if you want the have the same icon for your new app, open the file info of both apps (the original telegram and yours) with `CMD`+`I` and click on the icon in the top left of the original application. Hit copy `CMD`+`C`, click on yours and hit past `CMD`+`V`.


Please, someone add a guide for Windows.

**Comment by ShadowJonathan** [comment]
There should not be a workaround for this, this should be implemented directly into the client.

**Comment by Nachtalb** [comment]
@ShadowJonathan I think so too. But it is not implemented yet and I don't know C++ and so cannot make a PR. So for the time being until someone implements it, this is the best solution we've got IMHO.

**Comment by Banginpadr** [comment]
> @ShadowJonathan I think so too. But it is not implemented yet and I don't know C++ and so cannot make a PR. For the time being, until someone implements it, this is the best solution we've got IMHO.

exactly so i dont even know or see the reason to down vote my solution for this. people are just ungraceful

**Comment by hexauniverse** [comment]
@ShadowJonathan Multiple instances and multiple accounts are definitely different things, and look at 
the "actual behaviour" part

**Comment by ShadowJonathan** [comment]
@hexauniverse in this case, a strategy with multiple instances has to be used to have telegram work with multiple accounts at once, it is a workaround

**Comment by ShadowJonathan** [comment]
Telegram Mobile Android also now implements this natively, before this, only Telegram X did.

I think it's time for telegram desktop to also have this feature, to normalize the features all across the native clients, what do you think, @john-preston?

**Comment by fredboy** [comment]
I can't understand, why it's still not implemented. It's pretty important feature for a lot of people and the official android app supports it, so the devs ignoring this seems strange.

**Comment by BijoRino** [comment]
UNIGRAM

**Comment by RememberTheAir** [comment]
@fredboy simply because it's hard to implement it

**Comment by fredboy** [comment]
@RememberTheAir is it? Just create separate config folders and make a UI for it

**Comment by RememberTheAir** [comment]
@fredboy it's not just a matter of creating different dirs and configs and copying code from other softwares lol. TDesktop is a quiet big C++ codebase which has not been written with multiple accounts in mind and, for what I know, a lot of parts of the app (especially messages and media storage) would have to be rewritten to handle properly the activity of multiple accounts. That's not something trivial you do in a week, especially if you consider the frequency at which Telegram releases new features

**Comment by fredboy** [comment]
@RememberTheAir damn, i'm stupid. Anyway, easy or not, I hope they're working on it, cause it's quite useful and demanded feature.

**Comment by MaxMls** [comment]
I hope this task will be closed soon.

**Comment by VanToai** [comment]
Multiple accounts support is something we'll be implementing in [Telefuel](https://www.telefuel.com) (along with Discord-style chat folders & workspaces for Telegram, and more). 

If anyone's interested, we'd love to have you in the Telefuel beta. [Request an invitation](https://telefuel.com) or join our [Telegram community](https://t.me/telefuel) to learn more.

**Comment by Kogoruhn** [comment]
Just do it! :D
Please.

**Comment by levicki** [comment]
Why are PC users second-class citizens compared to mobile device users nowadays?

We shouldn't be waiting two years to get this basic feature in the official Telegram desktop client.

In my opinion the easiest way to implement it would be to change the list of chats into a tree with accounts represented as collapsible/expandable root nodes and chats as their leaf nodes so you can be logged in into more than one account at the same time from a single Telegram instance and window.

**Comment by ScottRFrost** [comment]
I tried https://github.com/UnigramDev/Unigram and realized it was kind of unstable and slow despite my decent system specs, so I used the power of GitHub and noticed there are THOUSANDS of forks of this project.  https://github.com/Forkgram/tdesktop worked fine for my 2nd account.  Certainly easier than trying to do command line arguments and run multiple instance of TDesktop (with the same icon, etc).

**Comment by amirashabani** [comment]
@ScottRFrost No Linux support? It appears Linux users are third-class citizens.

**Comment by RememberTheAir** [comment]
@amirashabani ...or maybe its developer is a Windows users and he's building that client based on his needs. I don't see any issue asking for Linux support, maybe you can try to ask to its dev if he's available to build for linux too

**Comment by ScottRFrost** [comment]
@amirashabani - If you're on ChromeOS (which is what I assume you mean by desktop linux) then you can just use the Android Telegram X, which support multiple accounts.

If you mean old school try-hard KDE or something, then you're out of luck, but you're probably used very few apps working if you're running old school desktop Linux.  Re-roll to Windows or Hackintosh.

**Comment by dausruddin** [comment]
@ScottRFrost Am I missing something? I can't find multi-account feature in Forkgram.

**Comment by RememberTheAir** [comment]
@dausruddin there isn't. He's suggesting to use different forks each one with its own account. Actually you can use tdesktop to do this, it can be launched multiple times using different working directories where config files are stored independently (see the OP)

**Comment by dausruddin** [comment]
@RememberTheAir oh. I am quite dissapointed. Thanks. I guess I'll stick to Unigram for now even I got error popping up sometimes.

**Comment by amirashabani** [comment]
@ScottRFrost Since when "KDE or something" is old school? Anyway, I'm using Arch Linux with Cinnamon as DE. But that's irrelevant; If they add this feature to the desktop client, I'll be able to use it, just like I'm using [this desktop client](https://desktop.telegram.org/changelog#beta-version) perfectly fine right now.

**Comment by MairwunNx** [comment]
+1 support multiple account it would be nice

**Comment by Dmitrij-Polyanin** [comment]
+1

**Comment by MairwunNx** [comment]
@john-preston Please pay attention and answer the open issue.
A whole year has passed, there is a desire to do this? 😉

**Comment by john-preston** [comment]
@MairwunNx There is a desire, there is no time :(

**Comment by Akshay-Rana-Gujjar** [comment]
**+1** multiple accounts feature

**Comment by albertchon** [comment]
+1 truly needed feature

**Comment by tvorogme** [comment]
+1 multiple accounts feature
--

**Comment by VanToai** [comment]
Telefeul (an unofficial Telegram client geared towards professionals & power users) recently announced support for multiple Telegram accounts in their web and desktop apps (Mac, Windows, and Linux):

[https://www.telefuel.com/blog/multiple-account-support](https://www.telefuel.com/blog/multiple-account-support)

**Comment by lastweakness** [comment]
> Telefeul (an unofficial Telegram client geared towards professionals & power users) recently announced support for multiple Telegram accounts in their web and desktop apps (Mac, Windows, and Linux):
> 
> https://www.telefuel.com/blog/multiple-account-support

Could you shut up and stop advertising your proprietary garbage?

**Comment by ScottRFrost** [comment]
@VanToai no one here supporting OPEN SOURCE Telegram wants to pay $10/mo for your proprietary, closed source knock-off.

Click his username then on the left column, click Block or report user, then Report Abuse and in the comment mention he's spamming his commercial product.

**Comment by VanToai** [comment]
Hi @lastweakness @ScottRFrost - I apologize that my message offended you. Our genuine intention with Telefuel is to create new, _unique_ value for our users and for the broader Telegram ecosystem.

You are not my enemy. I hope I am not yours. I wish you all the best.

**Comment by unidentified-user** [comment]
Very useful feature. I look forward to the implementation.

**Comment by unidentified-user** [comment]
**+1** Very useful feature. I look forward to the implementation.

**Comment by B1Z0N** [comment]
Contributing guidelines tell:

> Unfortunately we do not merge any pull requests that have new feature implementations, translations to new languages and those which introduce any new user interface elements.

If someone PRs this feature, will it be merged? I'm quite interested in this right now.

**Comment by Aokromes** [comment]
@B1Z0N preston stated multiple times memory leaks needs to be fixed 1st.

**Comment by john-preston** [comment]
@B1Z0N I don't think anyone will PR this, because it is too huge of a task.

**Comment by B1Z0N** [comment]
@john-preston , if no one will PR this, then how are we going to close the issue?

**Comment by B1Z0N** [comment]
@Aokromes , oops, sorry. I haven't seen that.

**Comment by RememberTheAir** [comment]
> @john-preston , if no one will PR this, then how are we going to close the issue?

Wait for Telegram to implement it

**Comment by ShadowJonathan** [comment]
> preston stated multiple times memory leaks needs to be fixed 1st.

> I don't think anyone will PR this, because it is too huge of a task.

Is multiple account support *possible*? And if so, is there demand for it? And if that's true, how high is it on the priority list according to that demand? 
Because these last two replies are contradicting eachother while assuming different answers of those questions.

**Comment by RememberTheAir** [comment]
> Is multiple account support possible?

I guess so, probably after a major refactoring of tdesktop's storage management, I assume based on what has been said by the developer until now

> And if so, is there demand for it?

Definitely

> And if that's true, how high is it on the priority list according to that demand?

Probably lower than people would expect since there's a workaround to use multiple accounts on the same machine and (I guess) the dev has to keep up with the mobile and wallet updates. Don't quote me on that. Just guessing

> Because these last two replies are contradicting eachother while assuming different answers of those questions.

The original question was whether tdesktop would accept a PR implementing multiple accounts support or not. This is a pretty huge task, as said by Preston, so a PR fixing the memory leaks problems and implementing multiple accounts is highly unlikely. I would say that, if done, right, they would accept it, but I'm pretty sure no one will submit it. So the only way to close this issue, that is, having multiple accounts support on tdesktop is wait for Telegram to put some resources on this task

**Comment by fgallaire** [comment]
Please !!

**Comment by ArmeF97** [comment]
> The original question was whether tdesktop would accept a PR implementing multiple accounts support or not. This is a pretty huge task, as said by Preston, so a PR fixing the memory leaks problems and implementing multiple accounts is highly unlikely to come. I would say that, if done right, they would accept it, but I'm pretty sure no one will submit it. So the only way to close this issue, that is, having multiple accounts support on tdesktop, is to wait for Telegram to put some resources on this task

It seems a job for @23rd  and ForkGram (@23rd  it's one of the most contributor of this repo)

**Comment by paradox70** [comment]
I solved multiple accounts for myself  in linux distro (ubuntu):
At the first create multiple workdir:
```
mkdir ~/Telegram/username1
mkdir ~/Telegram/username2

```
Then open `~/.bashrc` and insert alias for every account:
```
alias telegram_username1='/PATH/TO/DIR/Telegram -workdir /home/<USERNAME>/Telegram/username1/ -- %u'
alias telegram_username2='/PATH/TO/DIR/Telegram -workdir /home/<USERNAME>/Telegram/username2/ -- %u'

```
Now you can start multiple telegram by terminal:
```
# telegram_username1
# telegram_username2
```

**Comment by vvbv** [comment]
+1 truly needed feature

**Comment by andersonpem** [comment]
+1 for this one. It'd be really useful :)

**Comment by ynameit** [comment]
+1 would love to see this feature on windows client

**Comment by RememberTheAir** [comment]
I love to be notified by comments that can just be a 👍 reaction to the OP and adds nothing to the conversation

**Comment by fgallaire** [comment]
It's a more active lobbying

**Comment by AlexWayfer** [comment]
> It's a more active lobbying

In my experience, it doesn't matter. You can look at Microsoft's product issue, for example: https://github.com/microsoft/vscode/issues/519

4 years from the start, top-3 issue by 👍 reaction on the OP, hundreds of comments, and… nothing.

Such products have closed road-maps and plans. For example, Telegram is releasing new polls right now. For Telegram, it matters more than this issue. Developers of Telegram Desktop are busy for in-time release. So… They know about your wish, here are comments from maintainers, but there is no time unfortunately for implementing.

**Comment by sprite-1** [comment]
I just gave up and started using Unigram. It's not as polished but it's not the buggy mess it used to be, plus I get the added bonus of desktop access to Instant View and Secret Chats.

**Comment by ArmeF97** [comment]
> I just gave up and started using Unigram. It's not as polished but it's not the buggy mess it used to be, plus I get the added bonus of desktop access to Instant View and Secret Chats.

Well, on Windows is possible. Not everyone use windows :wink:

**Comment by nguyenkien** [comment]
> > I just gave up and started using Unigram. It's not as polished but it's not the buggy mess it used to be, plus I get the added bonus of desktop access to Instant View and Secret Chats.
> 
> Well, on Windows is possible. Not everyone use windows

He didn't say everyone switch to it.

**Comment by karmicdude** [comment]
+1 just do it

**Comment by dungmv** [comment]
wait for a long time

**Comment by minutesinch** [comment]
still waiting....

**Comment by revathskumar** [comment]
I am using 2 accounts in 2 different instances of telegram-desktop on Ubuntu. 

1. Installed via [snap](snapcraft.io/) `snap install telegram-desktop --edge`
2. Downloaded `tsetup` binary from [releases](https://github.com/telegramdesktop/tdesktop/releases).

Works fine for me.

**Comment by Mikaela** [comment]
However you will have duplicated resource usage (which is the same problem with profiles also) while the Android client runs just one instance which I would expect Telegram desktop to also do.

**Comment by devcline** [comment]
must have

**Comment by painhardcore** [comment]
This is so frustrating. It's a core feature of TG messenger. Why do ignore this issue so long?

**Comment by Aokromes** [comment]
> This is so frustrating. It's a core feature of TG messenger. Why do ignore this issue so long?

https://github.com/telegramdesktop/tdesktop/issues/4261#issuecomment-557918018

**Comment by devcline** [comment]
> > This is so frustrating. It's a core feature of TG messenger. Why do ignore this issue so long?
> 
> [#4261 (comment)](https://github.com/telegramdesktop/tdesktop/issues/4261#issuecomment-557918018)

its a normal for telegram,  they dont just implement features, they think how it would be better to implement them. We waited a pinned chats limit since 2017 and they made a "groups". I think they have some internal management that decides what is critical and what is not.

**Comment by Hypfer** [comment]
Since telegram is a free service, I don't think that it is acceptable to demand anything from the devs.

It _is_ an open source project so everyone could add this feature. `git clone` and off you go

**Comment by devcline** [comment]
> Since telegram is a free service, I don't think that it is acceptable to demand anything from the devs.
> 
> It _is_ an open source project so anyone could add this feature. `git clone` and off you go

"open source" is a wall between server side limits and users.

**Comment by dausruddin** [comment]
> It is an opensource project so anyone could add this feature. git clone and off you go

Thanks. I now have the source locally. I just need to learn how to code. Really helpful :D

**Comment by Hypfer** [comment]
@dausruddin well you can always hire someone to do what you can't or won't do.

**Comment by dausruddin** [comment]
@Hypfer Oh. I thought anyone could do this. Thanks. I will consider :)

**Comment by Aokromes** [comment]
> > > This is so frustrating. It's a core feature of TG messenger. Why do ignore this issue so long?
> > 
> > 
> > [#4261 (comment)](https://github.com/telegramdesktop/tdesktop/issues/4261#issuecomment-557918018)
> 
> It's a normal for telegram, they don't just implement features, they think how it would be better to implement them. We waited a pinned chats, limit since 2017 and they made a "groups". I think they have some internal management that decides what is critical and what is not. (or maybe its their business plan to make people wait)

https://github.com/telegramdesktop/tdesktop/issues/4261#issuecomment-544142079
https://github.com/telegramdesktop/tdesktop/issues/4261#issuecomment-557921243

**Comment by erfantkerfan** [comment]
This is so frustrating that this feature is not available after these many requests.
+1

**Comment by Aokromes** [comment]
read the message over your message.

**Comment by ArmeF97** [comment]
https://github.com/telegramdesktop/tdesktop/commit/b6ac4a0233cfc8ff6bda8f4aa8cd6463e38be9e7

👀

**Comment by cobalto** [comment]
Oh boy.
Who said dreams don't come true (sometimes)? 🙃

**Comment by AlexWayfer** [comment]
@Aokromes closed this 34 minutes ago

Sorry, but why? I don't see real code changes for multi-accounts.

**Comment by Aokromes** [comment]
> > @Aokromes closed this 34 minutes ago
> 
> Sorry, but why? I don't see real code changes for multi-accounts.

https://github.com/telegramdesktop/tdesktop/commit/b6ac4a0233cfc8ff6bda8f4aa8cd6463e38be9e7

**Comment by RememberTheAir** [comment]
Maybe this should've been closed only once the release is ready. Is it going to be released soon? Since we have a closed alpha there's still tests going on and maybe we will get it in the next days only

**Comment by AlexWayfer** [comment]
> b6ac4a0

I saw it, but there is "6 changed files with 15 additions and 19 deletions". It doesn't seem like real multi-account support.

> (ofc that's only the final commit of the work on multiaccount)

OK, then thank you all very much!

> Maybe this should've been closed only once the release is ready. Is it going to be released soon? Since there's a closed alpha, there's still tests going on and maybe we will get it in the next days only

There are a few different strategies for closing issues in project. As I've understand, in this project issues closed when merged in a main branch. So, let's just wait for a release.

**Comment by dungmv** [comment]
miracle

**Comment by MrTruth0** [comment]
> > [b6ac4a0](https://github.com/telegramdesktop/tdesktop/commit/b6ac4a0233cfc8ff6bda8f4aa8cd6463e38be9e7)
> 
> I saw it, but there is "6 changed files with 15 additions and 19 deletions". It doesn't seem like real multi-account support.

Yeah, it's normal, this is the "Closed Alpha", with this commit we can see that they're working on :3

**Comment by Aokromes** [comment]
> > > [b6ac4a0](https://github.com/telegramdesktop/tdesktop/commit/b6ac4a0233cfc8ff6bda8f4aa8cd6463e38be9e7)
> > 
> > 
> > I saw it, but there is "6 changed files with 15 additions and 19 deletions". It doesn't seem like real multi-account support.
> 
> Yeah, it's normal, this is the "Closed Alpha", with this commit we can see that they're working on :3

not anymore closed alpha. it was pushed to dev branch, so public.

**Comment by ange007** [comment]
> > > > [b6ac4a0](https://github.com/telegramdesktop/tdesktop/commit/b6ac4a0233cfc8ff6bda8f4aa8cd6463e38be9e7)
> > > 
> > > 
> > > I saw it, but there is "6 changed files with 15 additions and 19 deletions". It doesn't seem like real multi-account support.
> > 
> > 
> > Yeah, it's normal, this is the "Closed Alpha", with this commit we can see that they're working on :3
> 
> not anymore closed alpha. it was pushed to dev branch, so public.

How can this be tested?

**Comment by Aokromes** [comment]
> > > > > [b6ac4a0](https://github.com/telegramdesktop/tdesktop/commit/b6ac4a0233cfc8ff6bda8f4aa8cd6463e38be9e7)
> > > > 
> > > > 
> > > > I saw it, but there is "6 changed files with 15 additions and 19 deletions". It doesn't seem like real multi-account support.
> > > 
> > > 
> > > Yeah, it's normal, this is the "Closed Alpha", with this commit we can see that they're working on :3
> > 
> > 
> > not anymore closed alpha. it was pushed to dev branch, so public.
> 
> How can this be tested?

compiling yourself or waiting for release on the next few hours-days.

**Comment by gandalfsaxe** [comment]
Does work on https://web.telegram.org/ also?
